Cargo.toml

```diff
@@ -104,6 +106,11 @@ default-members = [
     "testing/separated/*/",
 ]
 
+[workspace.dependencies]
+# Use the following line to test local changes to UniFFI (and comment out the main one):
+# uniffi = { path = "../uniffi-rs/uniffi" }
+uniffi = "0.23"
```

Comment:
I'm pretty sure the change away from tilde version specs should make us accept any `0.23.X` version (https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio).

This means that if we make a patch release for `uniffi`, and take it on desktop, the a-s code doesn't need to be updated.